src/main/java/net/wurstclient/command/CmdList.java

```diff
@@ -64,6 +64,7 @@
 	public final UnbindCmd unbindCmd = new UnbindCmd();
 	public final VClipCmd vClipCmd = new VClipCmd();
 	public final ViewNbtCmd viewNbtCmd = new ViewNbtCmd();
+	public final DistCmd distCmd = new DistCmd();

```

Comment:
Hello thanks for these suggestions,
But as i said i am completely new to java 😅 before this i had some experience with python i mostly codded in that i just needed this function so i added this in wurst for myself but thought it could help others to so decided to do a pull request i could fix some problem but as i said am new to java so these all are unorganized and not properly formetted and stuff 😅 i just wrote it the easy way i could .. so if u want to cancel this its okay i am not really sure how to continue with this.
Sorry

Reply:
Hey! Never apologize! I was just offering suggestions seeing as you're new and I love helping out new talent wherever I can. Don't get discouraged by my comments, even if they look intimidating. 

If you want, I can even help you through the suggestions and my thinking behind them.

Keep up the great work, because it will take you far life. ❤️